src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,117 @@
+import type { ReactElement, SetStateAction } from 'react'
+import {
+  StyledArrow,
+  StyledArrowBox,
+  StyledCarouselContainer,
+  StyledCurrentDot,
+  StyledDot,
+  StyledDotBox,
+  StyledImage,
+  StyledImageBox,
+  StyledSlider
+} from './style'
+import { useEffect, useState } from 'react'
+import { ICON } from '../../constants/icons'
+
+interface CarouselProps {
+  images: { url: string; id: number }[]
+  isArrow: boolean
+}
+
+const Carousel = ({ images, isArrow }: CarouselProps): ReactElement => {
+  const arrowRight = ICON.CHEVRON_RIGHT_40
+  const arrowLeft = ICON.CHEVRON_LEFT_40
+  const [translateValue, setTranslateValue] = useState<number>(0)
+  const [mouseDownClientX, setMouseDownClientX] = useState<number>(0)
+  const [mouseUpClientX, setMouseUpClientX] = useState<number>(0)
+  const [cursorOn, setCursorOn] = useState<boolean>(false)
+
+  const moveCurrent = (imageIndex: number): void => {
+    setTranslateValue((imageIndex - 1) * 60)
+  }
+  const moveRight = (): void => {
+    if (translateValue !== 60 * (images.length - 1)) {
+      setTranslateValue(prev => prev + 60)
+    } else {
+      setTranslateValue(0)
+    }
+  }
+
+  const moveLeft = (): void => {
+    if (translateValue !== 0) {
+      setTranslateValue(prev => prev - 60)
+    } else {
+      setTranslateValue(60 * (images.length - 1))
+    }
+  }
+
+  const onMouseDown = (e: { clientX: SetStateAction<number> }): void => {
+    setMouseDownClientX(e.clientX)
+    setCursorOn(true)
+  }
+
+  const onMouseUp = (e: { clientX: SetStateAction<number> }): void => {
+    setMouseUpClientX(e.clientX)
+    setCursorOn(false)
+  }
+  const handleTouchStart = (e: any): void => {
+    setMouseUpClientX(e.touches[0].clientX)
+    setCursorOn(true)
+  }
+  const handleTouchStartEnd = (e: any): void => {
```

Comment:
```suggestion
  const handleTouchStartEnd: TouchEventHandler<HTMLDivElement> = e  => {
```
요기도 any 피할 수 있을 것 같아요!